README.md

```diff
@@ -27,22 +27,22 @@ You can download a ready to use Docker image from our repository:
 docker pull quay.io/3scale/apicast:v2
 ```
 
-The 3scale gateway image requires one of the two environment variables:
+The 3scale gateway image requires one of two environment variables. The first option will pull the latest gateway configuration from the 3scale API Manager. The second points to a local configuration file which has already been downloaded from 3scale:
```

Comment:
Yes @vramosp will be adding that to terminology as part of On Prem roll out. The API Manager = System + Backend. It is still relevant to refer to Admin Portal but that really is the UI, and it does not make much send that you "pull configs from the Admin Portal".